timeless/reservations/views.py

```diff
@@ -15,6 +15,13 @@
 
 BP = Blueprint("reservations", __name__, url_prefix="/reservations")
 
+class ReservationsListView(views.ListView):
+    """ List the reservation """
+    model = Reservation
+    template_name = "reservations/list.html"
+    context_object_list_name = "reservations"
+
+ReservationsListView.register(BP, "/teste/")
```

Comment:
@ebercamargo I see strange url path, pls fix :)

Reply:
I adjusted this.